src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -0,0 +1,236 @@
+          setWorkflow(wf)
+        }
+      })
+
+    initialize()
+  }, [workflow]) // eslint-disable-line react-hooks/exhaustive-deps
+
+  /*
+   * Page render
+   */
+  const { end, failures, start, status, workflowLog, workflowName, submittedFiles: { workflow: wdl } = {} } = workflow
+
+  const restructureFailures = _.map(({ message, causedBy }) => ({
+    message,
+    ...(!_.isEmpty(causedBy) ? { causedBy: simplifyDidNotStartFailures(restructureFailures(causedBy)) } : {})
+  }))
+
+  const simplifyDidNotStartFailures = failuresArray => {
+    const filtered = _.filter(({ message }) => !_.isEmpty(message) && !message.startsWith('Will not start job'), failuresArray)
+    const sizeDiff = !_.isEmpty(failuresArray) ? failuresArray.length - filtered.length : 0
+    const newMessage = sizeDiff > 0 ? [{ message: `${sizeDiff} jobs were queued in Cromwell but never sent to the cloud backend due to failures elsewhere in the workflow` }] : []
+
+    return [...filtered, ...newMessage]
+  }
+
+  return div({ style: { padding: '1rem 2rem 2rem', flex: 1, display: 'flex', flexDirection: 'column' } }, [
+    div({ style: { marginBottom: '1rem', display: 'flex', alignItems: 'center' } }, [
+      h(Link, {
+        href: Nav.getLink('workspace-job-history', { namespace, name }),
+        style: { alignSelf: 'flex-start', display: 'inline-flex', alignItems: 'center', padding: '0.5rem 0' }
+      }, [icon('arrowLeft', { style: { marginRight: '0.5rem' } }), 'Job History']),
+      div({ style: { margin: '0 0.5rem' } }, ['>']),
+      h(Link, {
+        href: Nav.getLink('workspace-submission-details', { namespace, name, submissionId }),
+        style: { alignSelf: 'flex-start', display: 'flex', alignItems: 'center', padding: '0.5rem 0' }
+      }, [`Submission ${submissionId}`]),
+      div({ style: { margin: '0 0.5rem' } }, ['>']),
+      div({ style: Style.elements.sectionHeader }, [`Workflow ${workflowId}`])
+    ]),
+    _.isEmpty(workflow) ? centeredSpinner() : div({ style: { display: 'flex', flexWrap: 'wrap' } }, [
+      makeSection('Workflow Status', [
+        div({ style: { lineHeight: '24px' } }, [makeStatusLine(style => statusIcon(status, style), status)])
+      ]),
+      makeSection('Workflow Timing', [
+        table({ style: { marginTop: '0.3rem', lineHeight: '20px' } }, [
+          tbody([
+            tr([td({ style: styles.sectionTableLabel }, ['Start:']), start ? td([Utils.makeCompleteDate(start)]) : 'N/A']),
+            tr([td({ style: styles.sectionTableLabel }, ['End:']), end ? td([Utils.makeCompleteDate(end)]) : 'N/A'])
+          ])
+        ])
+      ]),
+      makeSection('Links', [
+        div({ style: { display: 'flex' } }, [
+          h(Link, {
+            ...Utils.newTabLinkProps,
+            href: `${getConfig().jobManagerUrlRoot}/${workflowId}`,
+            style: { padding: '0.6rem' },
```

Comment:
nit: If we put `display: 'flex'` here, we can remove the inner div wrapper (below also)